### PR TITLE
Update Pebble layer field from "default" to "startup"

### PR DESCRIPTION
**Pull request by benhoyt**
Matches Pebble change https://github.com/canonical/pebble/pull/20

### CODE REVIEW

File: test/test_pebble.py [hunk 1]
```diff
@@ -479,7 +479,7 @@ def test_dict(self):
         s = pebble.Service('Name 2', d)
         self.assertEqual(s.name, 'Name 2')
         self.assertEqual(s.description, 'The lazy quick brown')
-        self.assertEqual(s.default, 'Dee Fault')
+        self.assertEqual(s.startup, 'Start Up')
```
Comment (jameinel) [inline test/test_pebble.py:482]: so do we validate the content of the fields, or we assume that pebble will be checking that enabled/disabled vs arbitrary content?

**Review by jameinel — APPROVED** [review]
I'm assuming this is just following up on the Pebble changes for updating the field name.

File: test/test_pebble.py (hunk 1 above)
Comment (benhoyt) [reply test/test_pebble.py:482]: Yeah, for these fields we don't validate -- we assume that Pebble has already validated the configuration.

**Review by hpidcock — APPROVED** [review]
LGTM thank-you

File: test/test_pebble.py (hunk 1 above)
Comment (hpidcock) [reply test/test_pebble.py:482]: @benhoyt @jameinel we can look at doing more validation on these values in the python classes, but for now they are validated server side which should be sufficient.

**Pull request merged**